src/main/java/com/conveyal/datatools/manager/jobs/MergeFeedsJob.java

```diff
@@ -965,10 +1048,26 @@ private int constructMergedTable(Table table, List<FeedToMerge> feedsToMerge,
                             .toArray(String[]::new);
                         writer.write(headers);
                     }
-                    // Write line to table (plus new line char).
+                    // Write line to table.
                     writer.write(rowValues);
                     lineNumber++;
                     mergedLineNumber++;
+                    if ((table.name.equals("calendar")) && serviceIdsToCloneAndRename.contains(rowValues[keyFieldIndex])) {
```

Comment:
Add an inline comment about what this block of code is all supposed to accomplish.